### PR TITLE
Authorize food list

### PR DESCRIPTION
**Pull request by brytebee**
### In this PR, I have added the ability to destroy/delete a food if your are the owner/creator of the food.

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -12,6 +12,12 @@ def initialize(user)
...
         recipe.user_id == user.id
       end
+      can :destroy, Food do |food|
+        food.user_id == user.id
+      end
+      can :update, Food do |food|
+        food.user_id == user.id
+      end
       can :read, :all
     end
```
Comment (ookino) [inline app/models/ability.rb:23]: I'm not able to add new food. Kindly check your ability class and make the required changes

File: app/views/foods/_food.html.erb [hunk 2]
```diff
@@ -2,8 +2,10 @@
...
-  <td>
-    <%= button_to "Delete Food" ,"foods/#{food.id}", method: :delete, data:
-    {confirm: "Are you sure"}, :class => "btn btn-sm btn-danger"%>
-  </td>
+  <% if can? :destroy, food %>
+    <td>
+      <%= button_to "Delete Food" ,"foods/#{food.id}", method: :delete, data:
+      {confirm: "Are you sure"}, :class => "btn btn-sm btn-danger"%>
+    </td>
+  <% end %>
```
Comment (ookino) [inline app/views/foods/_food.html.erb]: Kindly put the `if can` statement inside the `<td>` tag to avoid breaking the table's style

**Review by ookino — CHANGES_REQUESTED** [review]
### Changes Required

Good job implementing all requirements. However, there are a few bugs that need to be fixed.
Kindly go through my comments and make the required changes.

File: app/models/ability.rb (hunk 1 above)
Comment (brytebee) [reply app/models/ability.rb:23]: Done!

File: app/views/foods/_food.html.erb (hunk 2 above)
Comment (brytebee) [reply app/views/foods/_food.html.erb]: Done!

**Comment by ookino** [comment]
### Approved ✅ 
Good Job completing all required tasks. 
There is nothing left to say other than it's time to merge 👏🏾

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a blank ReadMe. K**indly ensure to provide a detailed  ReadMe.**


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**